partiql-plan/src/main/kotlin/org/partiql/plan/Operators.kt

```diff
@@ -52,6 +52,8 @@ import org.partiql.spi.value.Datum
  * The [Operators] interface is a factory to instantiate operator implementations.
  *
  * There is only ONE factory method per operators – i.e. no overloads or defaults.
+ *
+ * TODO support defining the type along with the operator's fields.
```

Comment:
self-review: adding this TODO since the operators currently do not have a good way to specify a type other than calling `<operator>.setType(...)` afterwards.